### DISCUSSION TITLE
Delete user within https://studio.apicur.io/

### DISCUSSION THREAD

**Discussion by LukMas** [ApicurioArchive/apicurio-studio#2397] [Q&A] [answered]
Hi!

Is it possible to delete myself as user from the online version of apicurio? 
https://studio.apicur.io/dashboard

Actually I find no option regards this. 

Thank you for any help.

**Comment by EricWittmann** [comment, marked as answer]
Sorry, we don't have any user management capabilities for the eval "try live" version of Studio right now.  If you want to message me directly I could remove your account.

**Comment by LukMas** [comment]
Hi!

Yes, I've seen later.
Thank you for answering. And, yes, please, could you delete it?

Best regards,
Luca

Il mer 20 mar 2024, 18:05 Eric Wittmann ***@***.***> ha
scritto:
